src/Domain.LinnApps/Reports/PurchaseOrdersReportService.cs

```diff
@@ -73,14 +73,20 @@ public ResultsModel GetOrdersByPartReport(DateTime from, DateTime to, string par
                             continue;
                         }
 
-                        this.ExtractDetailsForPartReport(values, order, orderDetail, delivery);
+                        this.ExtractDetailsForPartReport(values, order, orderDetail, delivery, order.CurrencyCode);
                     }
                 }
             }
 
             reportLayout.SetGridData(values);
             var model = reportLayout.GetResultsModel();
 
+            foreach ((var row, int i) in model.Rows.Select((value, i) => (value, i)))
+            {
+                model.SetGridValue(i, 5, model.GetGridValue(i, 5), decimalPlaces: 2);
+                model.SetGridValue(i, 7, model.GetGridValue(i, 7), decimalPlaces: 2);
```

Comment:
setting decimal places bit I mentioned in standup

Reply:
yeah doesn't feel right